projects/EIP-x stateless AA.MD

```diff
@@ -0,0 +1,143 @@
+EIP-x, Xtreamly
+Stateless Account abstraction that scales Ethereum.
+
+## Motivation
+
+Ethereum's state grows rapidly, burdening consensus nodes with 35 GB state and 100 GB with proofs, increasing every six months. 
+The increasing state size hampers scalability, imposing storage and processing burdens on nodes. Participants pay one-time costs perpetually,
+raising economic concerns. Additionally, ERC-4337 wallets incur elevated gas costs (~42000 gas for basic operations) due to multiple storage read/write costs, business logic overhead, and log payments not required in EOAs. A stateless-based solution is essential to maintain  Ethereum's efficiency and long-term viability. Additionally, the light clients, which do not store the entire state but rely on simplified verification mechanisms, struggle to efficiently access and  validate the state data against the mainnet. The lack of a concise and constant-sized proof of the current state limits light clients' ability to interact with the blockchain seamlessly.
+Therefore we Propose a stateless Account abstraction model which doesn't force any change in the current Ethereum core but can solve the above problems at the account level. The advantage of having succint zk proof of the state is that it is possible to broadcast it across the nodes. Currently, to validate a block, light clients must request the state piece by piece, and this results in many light clients frequently burdening the full node altogether.
+
+
+## Project description
+
+The challenge at hand is to enable stateless verification of blocks in the blockchain system. The goal is to allow clients to verify the
+correctness of individual blocks without needing a full state. Instead, clients can rely on compact witness files generated by state-holding nodes, which contain the portion of the state accessed by the block along with proofs of correctness. This stateless verification offers several benefits, including supporting sharding setups and reducing reliance on validators. The state provider entity is at the core of stateless AA, providing a succinct and constant-sized proof of the current state. Light clients can effortlessly verify this proof against the mainnet, improving scalability, and reducing resource requirements for light clients. Users and applications can efficiently interact with the blockchain, relying on the security and integrity of zk-proofs to verify the state. This approach enables more participants to engage with the network without needing the computational resources and storage capacity required for full nodes.
```

Comment:
this is great, I would just add a sentence or two about specific goal - implementing stateless client/provider entity..